ruma-events/src/room/message.rs

```diff
@@ -544,6 +544,14 @@ pub struct VideoInfo {
     /// Information on the encrypted thumbnail file.  Only present if the thumbnail is encrypted.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub thumbnail_file: Option<Box<EncryptedFile>>,
+
+    /// The [BlurHash](https://blurha.sh) for this video.
+    ///
+    /// This uses the unstable prefix in MSC2448.
+    #[cfg(feature = "unstable-pre-spec")]
+    #[serde(rename = "xyz.amorgan.blurhash")]
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Same here.

Reply:
Hmm, I was matching the style for the above fields here.